sql/src/lib.rs

```diff
@@ -8,18 +8,40 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+extern crate ordered_float;
+extern crate mentat_core;
+
 use std::error::Error;
 
+use ordered_float::OrderedFloat;
+
+use mentat_core::TypedValue;
+
 pub type BuildQueryError = Box<Error + Send + Sync>;
 pub type BuildQueryResult = Result<(), BuildQueryError>;
 
+pub enum BindParamError {
```

Comment:
It would be nice to build this on top of #341, which provides a nice framework for these error types.